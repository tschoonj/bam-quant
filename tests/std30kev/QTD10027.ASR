File: C:\xrfdaten\2014KW30\chris\std30kev\std10027.spe
Start: 22.07.2014 10:03:25
Stop: 22.07.2014 10:05:05
BESSY: 298.7083719 299.0874336
IONI: 2.00469E-11 1.997158E-11
ScanMotor1 exp_x: 48.05 48.05
ScanMotor2 exp_y: 83.45 83.5
Live: 98035
Run: 100000
Dead: 2
ICR: 130
OCR: 0
Accumulated BESSY: 59652.694438
Accumulated Ioni: 3.99053775E-09
N Accumulation: 200
Normfactor:  5.1123128e+008
E_cal:      0.020000000
E_cal_offset:      0.000000
Comments: Co
exp_x 72.7203979492188 72.7204
exp_y 122.465805053711 122.4658
exp_z 53.8899993896484 53.89
exp_w 135 135
exp_Tisch 860.6 860.6000079968
dcm_theta 0 0
det_y 52.85 52.85
dmm_energy 35 35.0001
dcm_energy 23 -184976.74536672
S3_horSize 0.05 4.99999999999997E-02
S3_verSize 0.05 5.00000000000002E-02
$PEAKS:   
      1
  27   1      6.925        13.         6.       2.67
