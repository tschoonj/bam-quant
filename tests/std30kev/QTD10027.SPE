$DATA:
0 2000
22902
14843
6391
1855
396
52
3
0
0
0
0
0
0
0
0
0
3
10
14
13
15
12
18
14
6
9
11
7
9
15
14
9
12
16
8
11
6
10
5
10
10
10
8
5
11
9
6
9
5
9
7
13
8
9
11
8
9
5
9
8
9
6
4
13
5
10
9
5
8
10
6
5
7
9
7
10
6
10
8
8
9
4
7
1
7
8
5
9
10
10
3
2
5
7
10
6
8
7
7
6
7
7
8
6
8
7
2
8
3
11
8
3
11
3
7
7
8
5
3
8
9
3
7
8
7
12
6
6
9
11
9
7
11
6
6
7
6
10
1
5
6
7
6
5
9
8
4
4
8
7
15
11
1
9
4
11
10
7
9
4
3
8
2
6
4
4
6
3
7
3
10
3
8
14
7
3
7
9
7
5
2
7
5
5
5
5
7
4
8
4
8
3
7
4
3
2
5
3
6
2
5
3
0
3
2
4
5
2
2
5
1
4
7
4
1
1
2
3
3
4
2
3
2
0
0
2
2
1
5
0
2
1
4
3
1
2
5
2
1
2
1
2
3
1
0
1
0
1
1
3
1
0
3
2
3
1
1
0
0
1
3
1
0
0
2
1
1
0
3
2
0
2
1
3
1
0
0
0
1
0
3
2
1
1
0
0
1
2
1
1
1
1
2
3
2
1
1
1
1
3
2
2
0
2
1
1
2
1
0
0
0
2
0
1
1
2
7
6
5
0
5
4
0
1
1
1
0
1
1
0
0
0
4
0
2
3
4
4
2
3
13
5
10
12
10
8
8
6
6
2
2
1
0
1
1
0
1
1
1
1
2
1
3
5
0
0
0
2
2
2
9
2
7
16
17
13
17
19
23
19
22
23
13
9
7
6
2
3
0
0
1
1
0
1
0
2
0
1
2
1
2
2
0
1
1
0
0
0
1
1
1
0
0
0
3
0
2
1
0
0
0
0
0
2
1
2
0
0
1
0
1
1
1
0
0
2
0
2
0
0
0
2
1
1
1
1
0
1
1
1
0
1
0
1
0
1
1
1
0
0
0
0
1
2
0
3
0
2
0
0
3
0
1
2
1
1
0
1
0
1
0
0
0
1
0
2
0
0
0
0
0
1
1
1
2
0
3
1
0
1
2
2
0
0
2
1
3
2
0
2
0
0
1
0
2
0
0
0
2
1
1
2
1
1
1
2
2
0
2
2
1
1
0
0
0
4
5
3
2
1
1
0
1
1
1
1
3
2
0
4
2
1
1
1
1
4
1
2
3
1
1
0
0
1
0
0
1
0
1
2
1
1
2
2
3
2
1
1
1
2
1
3
1
3
2
1
2
0
2
2
2
2
0
3
0
2
5
1
0
1
3
2
0
3
5
2
0
3
1
0
3
0
2
3
6
2
1
1
2
0
4
3
5
3
3
4
3
4
1
4
7
3
5
4
0
1
2
7
1
3
4
3
5
2
3
3
3
1
1
4
1
0
5
2
1
3
1
2
4
1
1
1
3
5
2
2
3
4
3
2
5
4
6
5
1
4
1
3
4
3
2
4
2
4
2
6
3
3
3
3
2
3
2
7
3
3
3
3
1
2
2
3
5
4
2
3
3
5
1
5
1
1
2
4
3
3
4
4
4
1
3
3
9
7
2
2
4
3
2
5
4
5
5
7
8
2
5
5
2
3
1
3
4
8
6
3
2
5
4
1
5
6
9
5
4
5
4
8
3
4
3
6
3
7
3
5
6
6
7
12
15
19
23
42
43
56
68
68
77
86
82
73
71
69
59
43
44
31
19
18
8
8
9
6
6
3
5
1
6
5
7
6
10
1
4
4
3
5
0
6
6
3
4
0
6
4
6
7
5
7
6
6
3
0
4
5
5
2
4
5
1
6
5
6
5
4
4
4
6
6
4
6
3
8
6
4
7
3
4
3
2
3
4
6
4
1
6
4
3
8
7
2
10
8
3
10
9
9
11
12
16
14
21
14
10
15
11
21
8
17
10
9
14
3
5
10
6
4
7
3
12
7
8
8
6
6
5
4
4
3
5
0
5
1
3
4
3
3
4
5
4
7
1
5
5
2
6
8
2
5
3
4
4
2
2
6
4
3
3
5
2
7
4
4
4
0
2
3
5
2
2
4
3
4
4
2
4
2
6
5
9
4
2
4
0
4
2
4
5
3
4
0
2
3
4
5
3
4
2
6
7
1
5
6
9
1
5
2
3
4
5
2
5
2
1
6
4
2
2
1
2
5
0
2
5
3
4
6
2
3
6
3
7
8
6
7
6
2
4
1
8
2
5
8
2
2
4
2
5
3
4
3
7
2
5
4
4
5
3
7
3
1
3
3
8
4
2
1
2
9
2
9
3
4
5
3
4
5
4
1
2
5
2
2
4
4
4
0
1
3
5
5
2
3
2
1
1
2
4
5
4
0
2
3
0
1
2
3
3
5
4
7
2
3
0
4
6
4
3
1
4
3
4
1
2
3
4
4
1
4
1
4
3
3
0
5
2
3
2
0
2
6
1
2
0
4
3
5
2
5
2
5
7
5
4
4
5
3
4
2
2
5
4
2
2
1
2
1
2
1
2
4
3
2
5
1
6
3
1
2
3
1
2
4
3
2
0
2
5
2
4
4
3
2
5
0
2
1
4
2
5
1
1
3
1
0
1
2
4
0
2
2
5
3
2
1
3
6
3
1
2
5
1
1
4
4
0
2
1
5
3
2
2
3
4
1
0
2
1
2
1
4
0
2
2
1
2
0
3
4
3
2
3
1
4
3
4
0
3
3
0
1
1
5
8
3
7
6
8
5
5
10
7
11
8
8
6
11
16
14
12
14
11
13
9
6
7
7
3
5
5
5
2
1
6
1
3
4
1
2
1
3
0
4
2
0
3
7
3
1
3
0
3
1
1
3
5
2
3
3
2
0
2
1
3
2
1
3
4
1
4
3
3
4
6
4
2
4
1
0
4
4
2
0
3
0
2
2
2
2
1
3
2
1
1
7
0
1
4
2
1
1
1
2
1
2
5
0
5
5
4
3
2
1
2
5
3
6
4
5
4
2
3
3
2
2
2
7
5
4
6
6
4
2
2
3
4
4
4
2
5
2
1
1
1
2
4
4
2
2
3
2
1
3
5
2
2
4
2
1
3
3
3
1
1
1
2
4
1
3
4
6
8
6
7
4
6
4
5
3
6
5
13
5
5
6
4
2
4
4
4
2
3
6
3
5
5
5
3
5
6
6
9
5
7
5
3
1
2
6
4
9
1
4
2
8
1
2
6
5
4
4
1
3
2
3
3
3
3
7
10
5
3
1
5
3
1
4
3
7
5
2
5
4
4
5
5
3
5
3
7
5
4
3
5
3
4
4
5
11
8
7
2
4
7
3
5
12
7
3
12
5
7
7
4
9
8
5
3
5
7
4
9
5
9
7
8
9
7
7
6
5
5
6
12
6
9
8
12
16
8
10
8
10
16
15
11
9
12
11
10
8
17
10
18
9
11
19
12
10
8
13
20
11
13
19
16
9
13
13
16
7
13
13
13
18
12
15
19
23
20
23
24
17
13
11
25
19
10
13
19
20
13
16
14
27
13
18
33
33
24
26
21
30
18
26
26
25
36
30
34
30
45
31
26
30
29
23
40
40
42
32
60
30
53
30
37
34
30
42
44
39
34
37
29
44
42
36
39
35
38
48
32
38
49
32
38
30
36
30
48
43
38
31
36
36
34
28
32
35
24
45
33
37
22
22
38
27
28
27
34
40
33
28
28
30
32
28
37
26
26
25
29
21
27
23
25
23
20
26
32
28
22
41
20
27
20
17
29
21
15
25
22
18
23
29
21
17
25
24
15
29
17
19
25
20
26
23
16
24
16
21
23
20
18
20
16
22
25
33
34
30
33
28
42
43
52
59
54
50
57
70
49
69
67
77
50
61
68
52
62
61
55
50
62
42
50
62
48
53
60
40
45
34
32
25
38
28
28
19
17
26
22
16
9
6
9
4
6
8
3
5
1
1
1
2
0
3
1
1
0
1
0
0
1
0
0
0
0
1
0
0
0
1
0
0
0
0
0
0
0
1
0
0
0
1
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
File: C:\xrfdaten\2014KW30\chris\std30kev\std10027.spe
Start: 22.07.2014 10:03:25
Stop: 22.07.2014 10:05:05
BESSY: 298.7083719 299.0874336
IONI: 2.00469E-11 1.997158E-11
ScanMotor1 exp_x: 48.05 48.05
ScanMotor2 exp_y: 83.45 83.5
Live: 98035
Run: 100000
Dead: 2
ICR: 130
OCR: 0
Accumulated BESSY: 59652.694438
Accumulated Ioni: 3.99053775E-09
N Accumulation: 200
Normfactor:  5.1123128e+008
E_cal:      0.020000000
E_cal_offset:      0.000000
Comments: Co
 
exp_x 72.7203979492188 72.7204
exp_y 122.465805053711 122.4658
exp_z 53.8899993896484 53.89
exp_w 135 135
exp_Tisch 860.6 860.6000079968
dcm_theta 0 0
det_y 52.85 52.85
dmm_energy 35 35.0001
dcm_energy 23 -184976.74536672
S3_horSize 0.05 4.99999999999997E-02
S3_verSize 0.05 5.00000000000002E-02
$PEAKS:   
      1
  27   1      6.925        13.         6.       2.67
