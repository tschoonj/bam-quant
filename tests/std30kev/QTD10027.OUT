     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ  AXIL IBM-PC V3.00                       10-08-2014   14:28:46  บ   
     บ  Spectrum: QTD10027.SPE                                     0.s บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ  Fitting Region: channels 130 -1800        3 iterations done    บ   
     บ  ChiSquare =     5.8   last change =   .51%    lambda= 0.E+00   บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ                        CALIBRATION  DATA                        บ   
     ฬออออออออออออออออหออออออออออออออออออออออออหอออออออออออออออออออออออน   
     บ                บ   Initial estimate     บ    Final estimate     บ   
     ฬออออออออออออออออฮออออออออออออออออออออออออฮอออออออออออออออออออออออน   
     บ  ZERO (eV)     บ      -44.6 ๑   100.0   บ     -44.6 ๑    97.2   บ   
     บ  GAIN (eV/ch)  บ     20.486 ๑   2.000   บ    20.486 ๑    .291   บ   
     บ det NOISE (eV) บ      314.6 ๑    40.0   บ     314.6 ๑    42.5   บ   
     บ  FANO factor   บ       .097 ๑    .050   บ      .097 ๑    .049   บ   
     ศออออออออออออออออสออออออออออออออออออออออออสอออออออออออออออออออออออผ   
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ                           PEAK  DATA                            บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ  # Line       E(KeV)  rel. int.   peak area   st. dev    chi-sq บ   
     บ               chan#   fwhm (eV)      backgr            tot. abs บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ                                                                 บ   
     บ  1 Co-Ka                                13. ๑      6.           บ   
     บ    KA1         6.925   1.00000          13. ๑      7.      2.7  บ   
     บ              340.215    336.63          25.           9.34E-01  บ   
     บ                                                                 บ   
     บ  2 Co-Kb                                27. ๑      7.           บ   
     บ    KB1         7.649   1.00000          27. ๑      9.      6.3  บ   
     บ              375.556    338.85          28.           9.50E-01  บ   
     บ                                                                 บ   
     บ  3 Pile_up                               0. ๑      0.           บ   
     บ                                                                 บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
                                                                           
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ                         FILTER BACKGROUND                       บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
                                                                           
